docs/features/cdc/cdc-preimages.rst

```diff
@@ -100,7 +100,7 @@ returns:
 
 The difference, compared to the previous example, is that the ``v1`` column has a value in all preimage rows: even though the second and third updates did not modify this column, the preimage shows its previous value anyway since we enabled full preimages.
 
-Preimage rows are only created for rows modified using inserts, updates, or row deletes. They are not created for range deletes or partition deletes. For example:
+Preimage rows are only created for rows modified using inserts, updates. <row deletes>. They are not created for range deletes. For example:
```

Comment:
The change to this sentence seems work in progress? It doesn't seem complete (or English) as-is.